tests/cisco_ios/show_ip_ospf_database_router/cisco_ios_show_ip_ospf_database_router.yml

```diff
@@ -0,0 +1,206 @@
+---
+parsed_sample:
```

Comment:
It appears the template still needs some work done. Comparing the `raw` and `yml` files, the LSA age appears to be off.

Reply:
Apologies - this should be fixed through 34dd6df.

I fixed this by introducing a stage change from the LSAs state to the LSAInfo state at the end of the LSAs state by triggering on and recording "LS age:". This is my first functional TextFSM template, and this feels like a bit of a hack to me. I am not sure if this is "acceptable" or not, so let me know if there's a better way to do this.

Reply:
This should work